### PR TITLE
`show` more things!

### PR DESCRIPTION
**Pull request by ericphanson**

### CODE REVIEW

File: src/count_loc.jl [hunk 1]
```diff
@@ -50,4 +50,3 @@ function loc_update!(d, key, new)
     d[key] = (; files = prev.files + 1, code = prev.code + new.code, comments = prev.comments + new.comments, blanks = prev.blanks + new.blanks )
 end
 
-count_julia_loc(table, dir) = sum(row.code for row in table if row.directory == dir && row.language == :Julia; init=0)
```
Comment (ericphanson) [inline src/count_loc.jl:53]: moved to the other file to be with the other counting functions

File: src/PackageAnalyzer.jl [hunk 2]
```diff
@@ -435,18 +448,22 @@ Package Pluto:
...
-  * number of contributors: 63
-  * has documentation: false
-  * has tests: true
+  * number of contributors: 73 (and 1 anonymous contributors)
+  * number of commits: 940
+  * has `docs/make.jl`: false
+  * has `test/runtests.jl`: true
   * has continuous integration: true
     * GitHub Actions
+
```
Comment (ericphanson) [inline src/PackageAnalyzer.jl:466]:
```suggestion
```

File: src/PackageAnalyzer.jl [hunk 3]
```diff
@@ -474,18 +491,21 @@ julia> using DataFrames
...
   * has license(s) in file: MIT
     * filename: LICENSE.md
     * OSI approved: true
-  * has documentation: true
-  * has tests: true
+  * has `docs/make.jl`: true
+  * has `test/runtests.jl`: true
   * has continuous integration: true
     * GitHub Actions
+
```
Comment (ericphanson) [inline src/PackageAnalyzer.jl:507]:
```suggestion
```

File: docs/src/index.md [hunk 4]
```diff
@@ -62,16 +67,18 @@ julia> using PackageAnalyzer
 
 julia> analyze(PackageAnalyzer)
 Package PackageAnalyzer:
-  * repo:
+  * repo:
```
Comment (giordano) [inline docs/src/index.md:70]: Why there is this extra space (here and other snippets elsewhere)? :thinking:

**Review by giordano — APPROVED** [review]
Looks good, only a comment about an extra space after `repo:` without repo

File: docs/src/index.md (hunk 4 above)
Comment (ericphanson) [reply docs/src/index.md:70]: That's how our `show` method works: `* repo: $(p.repo)`. If `p.repo == ""` then we get that space still. Not sure why it wasn't here before, I didn't touch that bit, just copy-pasted the new ones. Maybe we manually removed it before?

**Pull request merged**